examples/chatbot/index.ts

```diff
@@ -361,13 +361,10 @@ expressHandler.messageReceived(async (req, res) => {
       try {
         if (!wsUrl || !roomId) {
           // Create a room if not given as argument
-          const result = await fetch(`${WEBRTC_SERVER_BASE_URL}/rooms`, {
-            headers: {
-              Accept: 'application/json',
-              'Content-Type': 'application/json',
-            },
+          const result = await fetch(`${WEBRTC_SERVER_BASE_URL}/rooms/${randomUUID()}`, {
```

Comment:
Why did you need to specify this parameter? As far as I know, it works without it, doesn't it?